lab-keiran/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
if you install npm packages with the tag --save it will add them to the package.json,
so when you download the project again you can run `npm install` and it will install all the project's dependencies

`npm i --save <package-name>` for regular dependencies
and
`npm i --save-dev <package-name>` for dev dependencies like testing packages